NuRadioReco/modules/triggerTimeAdjuster.py

```diff
@@ -18,6 +18,9 @@ def __init__(self, log_level=logging.NOTSET):
         logger.setLevel(log_level)
         self.__sampling_rate_warning_issued = False
         self.begin()
+        logger.warning(f"triggerTimeAdjuster is deprecated and will be removed soon. In most cased you can safely delete the application\
+                       of this module as it is automatically applied in NuRadioMC simulations. If you really need to use this module, \
+                       please use the channelReadoutWindowCutter module instead.")
```

Comment:
1. using `\` this way does not lead to the right formatting - all the whitespace from the indentation is also included. To write the string over multiple lines you can just close and open the string (see e.g. the sampling rate warning).
2. This doesn't really matter for now, but can you change this to
```
import warnings

warnings.warn("Warning message", DeprecationWarning)
```
The reason for this is that this will allow us to raise errors from deprecation warnings in the tests (not yet implemented), which will make it easier to find and fix deprecated code used within NuRadioMC.

Reply:
fixed